### PR TITLE
clickhouse: add article how to connect to clickhouse with python

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/DOC-46

Our users have different backgrounds and work in different programming languages - now I'm adding a how-to article on connecting to a clickhouse service with python

### CODE REVIEW

File: docs/products/clickhouse/howto/connect-with-python.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+Identify connection information
+'''''''''''''''''''''''''''''''
+
+To run the code for connecting to your service, first identify values of the following variables:
+
+===========================     ======================================================================================
+Variable                        Description
+===========================     ======================================================================================
+``https://HOST:HTTPS_PORT``     ``Host`` and ``Port`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``query``                       Smaple query you want to run, for example ``SELECT 1``
```
Comment (kmichel-aiven) [inline docs/products/clickhouse/howto/connect-with-python.rst:61]:
```suggestion
``query``                       Sample query you want to run, for example ``SELECT 1``
```

File: docs/products/clickhouse/howto/connect-with-python.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+To connect to your Aiven for ClickHouse® service with Python, you can use either the native protocol or the HTTPS protocol. This article provides you with instructions for both the scenarios.
+
+Connect with the native protocol
+--------------------------------
+
+Pre-requisites
+''''''''''''''
+
+* `Python 3.6 or later <https://www.python.org/downloads/>`_
```
Comment (kmichel-aiven) [inline docs/products/clickhouse/howto/connect-with-python.rst:12]: For the native python driver, it's 3.5 or later.
```suggestion
* `Python 3.5 or later <https://www.python.org/downloads/>`_
```

File: docs/products/clickhouse/howto/connect-with-python.rst [hunk 3]
```diff
@@ -0,0 +1,95 @@
+Connect to the Aiven for ClickHouse® service with Python
+========================================================
+
+To connect to your Aiven for ClickHouse® service with Python, you can use either the native protocol or the HTTPS protocol. This article provides you with instructions for both the scenarios.
```
Comment (kmichel-aiven) [inline docs/products/clickhouse/howto/connect-with-python.rst:4]:
```suggestion
To connect to your Aiven for ClickHouse® service with Python, you can use either the native protocol or the HTTPS protocol. This article provides you with instructions for both scenarios.
```

File: docs/products/clickhouse/howto/connect-with-python.rst [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+Variable                        Description
+===========================     ======================================================================================
+``https://HOST:HTTPS_PORT``     ``Host`` and ``Port`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``query``                       Query you want to run, for example ``SELECT 1``
+``X-ClickHouse-Database``       ``Database Name`` available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**, for example ``system``
+``X-ClickHouse-User``           ``User`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``X-ClickHouse-Key``            ``Password`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``X-ClickHouse-Format``         Format for the output from your query, for example ``JSONCompact``
```
Comment (anelook) [inline docs/products/clickhouse/howto/connect-with-python.rst:65]: The available formats the same as in https://docs.aiven.io/docs/products/clickhouse/reference/supported-input-output-formats.html? 
BTW, on that note, the title for https://docs.aiven.io/docs/products/clickhouse/reference/supported-input-output-formats.html was changed to "Formats for ClickHouse-Kafka data exchange". I believed that these formats are not only Kafka related, but general ones. You can see here the article on the side of  ClickHouse docs - https://clickhouse.com/docs/en/interfaces/formats/

Comment (wojcik-dorota) [reply docs/products/clickhouse/howto/connect-with-python.rst:65]: @anelook , the title has been changed as requested in https://aiven.atlassian.net/browse/DOC-20 by @kmichel-aiven

Comment (kmichel-aiven) [reply docs/products/clickhouse/howto/connect-with-python.rst:65]: Yes :)

The formats available to connect ClickHouse with python are all of these: https://clickhouse.com/docs/en/interfaces/formats/
The formats available to connect ClickHouse with Kafka are only a small subset: the ones in https://docs.aiven.io/docs/products/clickhouse/reference/supported-input-output-formats.html

**Review by kmichel-aiven — APPROVED** [review]
LGTM

**Comment by wojcik-dorota** [comment]
> LGTM

@kmichel-aiven , could you please merge in that case?

**Pull request merged**